platforms/hyperledger-fabric/charts/catools/templates/configmap.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: orderers-script-cm
+  namespace: {{ .Values.metadata.namespace }}
+  labels:
+    app.kubernetes.io/name: orderers-script
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+data:
+  generate-crypto-orderer.sh: |-
```

Comment:
[nitpick] If we can load `ConfigMap` files via `Files.Glob` approach, it will help in quick debugging the `bash` scripts separately & also makes it easy to read/fix.